src/main/java/org/simulator/sbml/EquationSystem.java

```diff
@@ -1809,9 +1809,8 @@ public void propertyChange(PropertyChangeEvent propertyChangeEvent) {
    * @param highOrderEvents
    */
   protected void pickRandomEvent(List<Integer> highOrderEvents) {
-    int length = highOrderEvents.size();
-    int random = RNG.randomInt(0, length - 1);
```

Comment:
`randomInt` was the only method in `RNG` that was used anywhere.

`Random.nextInt(n)` does the same thing as `RNG.nextInt(0, n-1)`, so we can just use the `Random` API directly here.

`ThreadLocalRandom.current()` is simply a fast `Random` instance.